omero/sysadmins/server-setup-examples.rst

```diff
@@ -117,3 +117,11 @@ Network infrastructure
 The server is hosted inside the faculty of medicine where the network works at
 100 Mbit/s. There are are 4 Gbit/s ports on the server but only one is
 currently in use.
+
+.. seealso::
+
+    `IDR deployment page <https://idr.openmicroscopy.org/about/deployment>`_
+        Information about the deployment of the
+        `Image Data Resource <https://idr.openmicroscopy.org>`_ which hosts
+        over 10 million images - see
```

Comment:
reviewing: https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/server-setup-examples.html

I think I wouldn't use "see" here since it makes it seem like that's where the info has. Can "over 10 million images" simply link to the studies page if someone wants that context? I could also see not using the "seealso" box but having this as a header like the others.

Reply:
Thanks, I agree the see also box seems to refer to the previous section, a63032f rewrites the information as a new section to prevent the confusion